kong/db/migrations/core/005_120_to_130.lua

```diff
@@ -141,7 +141,7 @@ return {
             algorithm = "consistent-hashing"
           end
 
-          local _, err = connector:query([[
+          local _, err = coordinator:execute([[
```

Comment:
what is the difference between `connector:query` and `coordinator:execute`, and when we should use one or the other?

Reply:
connector:query may implicitly connect (and in cassandra it by default takes read-only connection).